integration-tests/src/test/java/org/web3j/protocol/core/ObservableIT.java

```diff
@@ -21,8 +21,8 @@
  */
 public class ObservableIT {
 
-    private static final int EVENT_COUNT = 5;
-    private static final int TIMEOUT_MINUTES = 5;
+    private static final int EVENT_COUNT = 1;
```

Comment:
You should leave these values as they were.
You can add the extra static variable for edge cases like the one you are testing. i.e.
`private static final int SINGLE_EVENT_COUNT = 1`
Then add specific tests for those edge cases.